tools/Tools.Common/Models/CmdletMetadata.cs

```diff
@@ -354,12 +354,17 @@ public override bool Equals(Object obj)
                 var otherParameterSet = other.ParameterSets.Find(p => string.Equals(p.Name, thisParameterSet.Name, StringComparison.OrdinalIgnoreCase));
                 if (otherParameterSet == null)
                 {
+                    // Console.WriteLine($"Parameter set {thisParameterSet.Name} in cmdlet {this.Name} is not found in new module.");
```

Comment:
Please remove those comment-out code.

Reply:
I don't want to remove them as it's useful if we need to debug version bumper in ADO pipeline.